config/config.apps.sample.php

```diff
@@ -53,6 +53,30 @@
  */
 'admin_audit.groups' => ['group1', 'group2'],
 
+/**
+ * App: Files Antivirus
+ *
+ * Possible keys: `files_antivirus.av_path` STRING
+ *
+ * Possible keys: `files_antivirus.av_cmd_options` STRING
+ */
+
+/**
+ * Default path to the _clamscan_ command line anti-virus scanner.
+ * Only necessary in case the operating mode of the `files_antivirus` app is set to executable mode.
+ * Do not change this option if the app operating mode is not set to executable.
+ * See the documentation for more details.
+ */
+'files_antivirus.av_path' => '/usr/bin/clamscan',
+
+/**
+ * Command line options for the _clamscan_ command line anti-virus scanner.
+ * Only necessary in case the operating mode of the `files_antivirus` app is set to executable mode.
+ * Do not change this option if the app operating mode is not set to executable.
```

Comment:
```suggestion
 * This setting only applies when the operating mode of the `files_antivirus` app is set to executable mode.
```
Maybe a sentence like this can replace both line 66 and 67?